homeassistant/components/device_tracker/luci.py

```diff
@@ -65,10 +58,24 @@ def scan_devices(self):
     def get_device_name(self, device):
         """Return the name of the given device or None if we don't know."""
         name = next((
-            result.name for result in self.last_results
+            result.hostname for result in self.last_results
             if result.mac == device), None)
         return name
 
+    def get_extra_attributes(self, device):
+        """
+        Get extra attributes of a device.
+
+        Some known extra attributes that may be returned in the dict
+        include Mac Address (mac), Network Device (dev), Ip Address
+        (ip), reachable status (reachable), Associated router
+        (host), Hostname if known (hostname) among others.
+        """
+        device = next((
+            result for result in self.last_results
+            if result.mac == device), None)
```

Comment:
We can't access keys in the dict as an attribute.

Reply:
result is no longer a dict. It is a named tuple since 1.0.0 of the openwrt-luci-rpc module.

Reply:
We talk about a dict in the docstring. I think that's need clarification then.

Reply:
Good point. Will fix the docstring 👌